### PR TITLE
LLM05-LLM10 ITA

### PR DESCRIPTION
**Pull request by fcilli**
# PR Checklist 🚨

It is intended that only defined "*vulnerability entry leads*" should be able to create PR's (See [CODEOWNERS](https://github.com/OWASP/www-project-top-10-for-large-language-model-applications/blob/main/CODEOWNERS) and [Core Team](https://github.com/OWASP/www-project-top-10-for-large-language-model-applications/wiki/Core-Team))

If you are not a CODEOWNER for the entry of artifact, please see our Issues and Discussions boards

## Proposed changes

Adding Italian version of LLM10_ModelTheft.md

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have x-referenced the appropriate issue this addresses within the [projects board](https://github.com/OWASP/www-project-top-10-for-large-language-model-applications/issues)
- [ ] I have applied the relevant [labels](https://github.com/OWASP/www-project-top-10-for-large-language-model-applications/labels) this PR addresses

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by mattbit** [comment]
Thanks @fcilli, will proofread later today.

File: 1_1_vulns/it/LLM10_ModelTheft.md [hunk 1]
```diff
@@ -1,49 +1,49 @@
...
+2. Limitare l'accesso dell'LLM alle risorse di rete, ai servizi interni e alle API.
+   1. Questa pratica è efficace per tutti gli esempi di attacchi comuni, fornendo copertura dai rischi e dalle minacce interne (insider threats), e limitando allo stesso tempo ciò a cui l'applicazione LLM "_ha accesso_", rappresentando dunque un meccanismo o metodo preventivo, per prevenire o interrompere attacchi laterali (side-channel attacks).
+3. Utilizzare un Inventario o Registro centralizzato per i modelli di Machine Learning (ML Models) utilizzati in produzione. Avere un registro di modelli centralizzato impedisce l'accesso non autorizzato ai modelli di ML tramite controlli di accesso, autenticazione e grazie alla capacità di monitoraggio/registrazione, elemnti fondamentali per il governo di questi processi (governance). Avere un repository centralizzato è inoltre vantaggioso per collezionare dati sugli algoritmi utilizzati dai modelli, ai fini della conformità, valutazione dei rischi e loro mitigazione.
```
Comment (mattbit) [inline 1_1_vulns/it/LLM10_ModelTheft.md:30]:
```suggestion
3. Utilizzare un Inventario o Registro centralizzato per i modelli di Machine Learning utilizzati in produzione. Avere un registro di modelli centralizzato impedisce l'accesso non autorizzato ai modelli di ML tramite controlli di accesso, autenticazione e grazie alla capacità di monitoraggio/registrazione, elementi fondamentali per la governance di questi processi. Avere un repository centralizzato è inoltre vantaggioso per raccogliere dati sugli algoritmi utilizzati dai modelli ai fini di conformità, valutazione dei rischi e loro mitigazione.
```

File: 1_1_vulns/it/LLM10_ModelTheft.md [hunk 2]
```diff
@@ -1,49 +1,49 @@
...
+4. Monitorare e verificare regolarmente i registri (log) di accesso e le attività relative agli archivi dei modelli LLM, per rilevare e rispondere tempestivamente a qualsiasi comportamento sospetto o non autorizzato.
+5. Automatizzare la distribuzion delle cosiddette MLOps (Gestione Operativa dei Modelli di Machine Learning) attraverso flussi di lavoro per la governance, il tracciamento e l'approvazione, per rafforzare i controlli di accesso e rilascio, all'interno dell'infrastruttura.
```
Comment (mattbit) [inline 1_1_vulns/it/LLM10_ModelTheft.md:32]:
```suggestion
5. Automatizzare la distribuzion delle cosiddette MLOps (Gestione Operativa dei Modelli di Machine Learning) attraverso flussi di lavoro per la governance, il tracciamento e l'approvazione, per rafforzare i controlli di accesso e rilascio all'interno dell'infrastruttura.
```

File: 1_1_vulns/it/LLM10_ModelTheft.md [hunk 3]
```diff
@@ -1,49 +1,49 @@
...
+6. Implementare controlli e strategie di mitigazione per ridurre e|o contenere il rischio di attacchi indiretti (side channel) causati da tecniche di iniezione di prompt (prompt injection).
+7. Limitare la frequenza delle chiamate API dove applicabile e/o utilizzare filtri per ridurre il rischio di esfiltrazione dati dalle applicazioni LLM, o implementare tecniche (ad es. sistemi di Data Loss Prevention) per rilevare attività di estrazione anche da altre fonti.
+8. Implementare un addestramento avversariale mirato alla resilienza, sia per aiutare a rilevare query di estrazione, e rafforzare le misure di sicurezza fisica.
```
Comment (mattbit) [inline 1_1_vulns/it/LLM10_ModelTheft.md:35]:
```suggestion
8. Implementare un addestramento avversariale mirato alla resilienza per aiutare a rilevare query di estrazione e rafforzare le misure di sicurezza fisica.
```

File: 1_1_vulns/it/LLM10_ModelTheft.md [hunk 4]
```diff
@@ -1,49 +1,49 @@
...
+9. Implementare un framework di marcatura modale e|o temporale (watermarking) nelle fasi di integrazione (input) e monitoraggio (output, trasformazione), parte del ciclo di vita di un LLM.
```
Comment (mattbit) [inline 1_1_vulns/it/LLM10_ModelTheft.md:36]:
```suggestion
9. Implementare un framework di marcatura modale e/o temporale (watermarking) nelle fasi di integrazione (input) e monitoraggio (output, trasformazione), parte del ciclo di vita di un LLM.
```

File: 1_1_vulns/it/LLM10_ModelTheft.md [hunk 5]
```diff
@@ -1,49 +1,49 @@
...
-4. A security control failure is present within the supply-chain and leads to data leaks of proprietary model information.
-5. A malicious attacker bypasses input filtering techniques and preambles of the LLM to perform a side-channel attack and retrieve model information to a remote controlled resource under their control.
+1. Un attaccante abusa una vulnerabilità nell'infrastruttura di un'azienda per accedere senza autorizzazione al loro repository di modelli LLM. L'attaccante procede quindi all'esfiltrazione di modelli LLM di valore e li utilizza per lanciare un servizio concorrente di elaborazione del linguaggio o estrarre forzatamente informazioni sensibili, causando gravi danni finanziari all'azienda proprietaria del modello.
+2. Un dipendente insoddisfatto divulga modelli o artefatti correlati. L'esposizione pubblica che questo scenario rappresenta, aumenta la conoscenza di attaccanti ulteriori, consentendo attacchi avversariali di tipo "gray box" (conoscenza parziale) o, in alternativa, consente di rubare direttamente la proprietà intellettuale esposta.
```
Comment (mattbit) [inline 1_1_vulns/it/LLM10_ModelTheft.md:41]:
```suggestion
2. Un dipendente insoddisfatto divulga modelli o artefatti correlati. L'esposizione pubblica che questo scenario rappresenta aumenta la conoscenza a disposizione di possibili attaccanti, consentendo attacchi avversariali di tipo "gray box" (conoscenza parziale) o, in alternativa, consente di rubare direttamente la proprietà intellettuale esposta.
```

File: 1_1_vulns/it/LLM10_ModelTheft.md [hunk 6]
```diff
@@ -1,49 +1,49 @@
...
+3. Un attaccante interroga l'API con input accuratamente selezionati e raccoglie un numero sufficiente di output per creare un modello ombra (shadow).
+4. Una mancanza, nei controlli di sicurezza della catena di fornitura (supply chain) porta a perdite di dati relativi a informazioni proprietarie del|sul modello.
```
Comment (mattbit) [inline 1_1_vulns/it/LLM10_ModelTheft.md:43]:
```suggestion
4. Una mancanza nei controlli di sicurezza della catena di fornitura (supply chain) porta a perdite di dati relativi a informazioni proprietarie del o sul modello.
```

File: 1_1_vulns/it/LLM10_ModelTheft.md [hunk 7]
```diff
@@ -1,49 +1,49 @@
...
+5. Un attaccante malintenzionato elude le tecniche di filtraggio degli input e negli stadi iniziali dell'LLM, per eseguire un attacco indiretto|laterale e recuperare informazioni sul modello verso una risorsa remota, sotto il suo controllo.
```
Comment (mattbit) [inline 1_1_vulns/it/LLM10_ModelTheft.md:44]: Not 100% sure what "preambles of the LLM" means here, but I would interpret it as the initial instructions given to the LLM (e.g. the part of the prompt preceding the user-provided text). Would that make sense?

```suggestion
5. Un attaccante malintenzionato elude le tecniche di filtraggio degli input e le istruzioni iniziali dell'LLM, per eseguire un attacco indiretto/laterale e recuperare informazioni sul modello caricandole su una risorsa remota sotto il suo controllo.
```

**Review by mattbit — COMMENTED** [review]
Thanks @fcilli, looks good!

I've left a few comments, mostly minor formatting and a few rephrasing suggestions.

**Comment by fcilli** [comment]
💪👍

https://calendly.com/fabrizio-cilli/45min

File: 1_1_vulns/it/LLM10_ModelTheft.md [hunk 8]
```diff
@@ -1,49 +1,49 @@
...
+1. Un aggressore sfrutta una vulnerabilità nell'infrastruttura di un'azienda per accedere senza autorizzazione al loro archivio (repository) di modelli LLM grazie a configurazioni errate nella sicurezza della rete o delle applicazioni.
+2. Uno scenario di minaccia interna in cui, un dipendente insoddisfatto, divulga modelli o artefatti correlati ai modelli.
+3. Un attaccante interroga l'API del modello, utilizzando input meticolosamente elaborati e tecniche di iniezione di prompt (comandi e parametri da linea di comando), per raccogliere un numero sufficiente di output (risposte del modello) utilizzandoli per creare successivamente un modello ombra (shadow model).
```
[the remark this answers is not in the document]
Comment (fcilli) [reply 1_1_vulns/it/LLM10_ModelTheft.md:13]: A simple recall of the command line nature of prompt injection, being it via API or frontend. Similarly to "LLM" acronym, was just a way to explain the concept. (could be superfluous)

File: 1_1_vulns/it/LLM10_ModelTheft.md (hunk 1 above)
Comment (fcilli) [reply 1_1_vulns/it/LLM10_ModelTheft.md:30]: I tried to follow the rule of "not using anglophone terms without at least a translation next to them".

**Comment by fcilli** [comment]
Reopening with latest files

**Pull request closed without merging**